Steer/include/Steer/O2RunSim.h

```diff
@@ -60,7 +60,7 @@ class O2RunSim : public FairRunSim
     fRootManager->InitSink();
 
     // original FairRunSim follows
-    FairGeoLoader* loader = new FairGeoLoader(fLoaderName->Data(), "Geo Loader");
+    FairGeoLoader* loader = new FairGeoLoader(fLoaderName.Data(), "Geo Loader");
```

Comment:
With `GetGeoModel()->Data()`  I guess it should work both with old and new versions, will give a try

Reply:
Good idea.